xml/installation-installation-ses_integration.xml

```diff
@@ -9,62 +9,29 @@
  <title>&ses; Integration</title>
  <para>
   The current version of &productname; supports integration with &ses;
-  (SES). Integrating &ses; enables &ceph; block storage as well as object and
-  image storage services in &productname;.
+  (SES). Integrating &ses; enables &ceph; block storage as
+  well as object and image storage services in &productname;.
  </para>
+ <para>
+  The following sections detail integration with the latest offering of &ses; and
+  previous releases.
+  For more information on &ses;, see
+  the <link xlink:href="https://www.suse.com/documentation/suse-enterprise-storage-5/">SES 5 and 5.5 documentation</link>
```

Comment:
SES 6 is missing? (right now GMC)

Reply:
In theory, yes? But it's not published: https://www.suse.com/documentation/ I'll check in with @tbazant

Reply:
Not published until June 24. Will update after that :)